samples/pmic/npmx/buck/README.rst

```diff
@@ -0,0 +1,101 @@
+.. _nPM1300 BUCK:
+
+nPM1300 BUCK
+############
+
+Overview
+********
```

Comment:
Sample is not matching the template: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/sample_README.html